src/components/DefineCompany/BusinessContactInfo.vue

```diff
@@ -6,14 +6,14 @@
         </v-flex>
         <v-flex md4>
           <div><label><strong>Email Address</strong></label></div>
-          <div id="lbl-email">{{ !!contact.email ? contact.email : 'Not entered'}}</div>
+          <div id="lbl-email">{{ !!contact.email ? contact.email : '(Not entered)'}}</div>
```

Comment:
Might need a space between `(Not entered)' and the closing }}

Reply:
The linter (and Vue) doesn't care about this but for the sake of consistency, I will update!

Reply:
Fixed.